### DISCUSSION TITLE
Getting to know Lethe users: Please introduce yourself and your research

### DISCUSSION THREAD

**Discussion by blaisb** [chaos-polymtl/lethe#667] [General]
As some of you may have noticed, the Lethe community has grown in the last few months and we now have users from outside of the Montréal area. If you are using Lethe and are not based at Polytechnique Montréal, I would be very interested in learning which features you are using, for what type of applications and if there is anything we could do to help you or begin collaborating with your research groups :).

I will start.
I'm Bruno Blais, I am Associate Professor at Polytechnique Montréal. I use Lethe for many things, but lately mostly for the study of granular flows (using DEM), coupled solid-fluid flows (using CFD-DEM) and I have been starting to work on free surface flows that include gravitational forces (e.g. sloshing) using the VOF model we are developing :).

**Comment by Wendi-L** [comment]
Hi Bruno,

Thanks for opening this Q&A.

I’m Wendi Liu, Senior Computational Scientist at the Science & Technology Facilities Council, UK. I am one of the key contributors of the partitioned code coupling framework [MUI](https://mxui.github.io/). I got to know Lethe from SoftwareX, and was amazed by this well-organised repository and documentation. I am not an official user of Lethe yet, but I can see its great potential for my research interests.

I am a part of a research team that focuses on high performance multiphysics and multiscale simulations by using high performance computing facilities. We are not from a chemical engineering background (ocean engineering for myself, also turbulence, aeronautics and computer science from our team). Our research interest includes fluid-structure interaction (such as wave and sloshing impact on elastic structure), turbulence modelling and high performance code coupling.

**Comment by DanW97** [comment]
Hi Bruno,

I'm Dan, an EngD student with the University of Birmingham, in conjunction with Johnson Matthey as an industrial sponsor. My research is primarily the modelling of stirred milling processes. I first heard of Lethe when searching for alternatives to CFDEM coupling and have since been using it for other stuff such as determining drag laws in packed spheres and evaluating the Nitsche IB method vs MRF in OpenFOAM. I have also written some tools concerning geometry creation that I will eventually share with the Lethe utils repo once I've got deal.II to play nice with Manjaro!

The research group I'm with is focused on modelling granular systems, particularly fluidised beds and stirred mills currently, and also characterisation via Positron Emission Particle Tracking (PEPT). We also *really* care about better and reproducible validation of models in general, so we have some freely available [tools](https://github.com/uob-positron-imaging-centre) to help with this.

I am also happy to try and contribute code to Lethe, particularly for the sharp solver, so that i can include a slightly complex impeller into my simulations. FEM is relatively new to me compared to finite volume methods, but if I am pointed in the right direction, I should be able to be useful!

Cheers,

Dan

**Comment by Majid-MHZ** [comment]
Hi Bruno,

I'm Majid, a Ph.D. candidate at Pusan National University of South Korea. I am doing a powder-forming compaction simulation using the DEM method by Abaqus, and I have been requested to perform this simulation in an open-source FEM. I found Lethe a great open-source in this regard, but I am literally new to this software and even don't know how it works. I have some questions for you. The questions are as below:

I want to know if it is possible to do the powder-forming compaction with Lethe software. As you may know, powder-forming compaction is done by importing particles (e.g. 1000 particles) into a box and applying pressure with a punch to compact them. I need to define a general contact between the particles and the box walls as well as between the particles themselves. ( Please see the attached file). Can you please let me know if the Lethe software can handle this type of simulation? 

I am really grateful if you could help me in this regard.

Best regards,
Majid
![1](https://github.com/chaos-polymtl/lethe/assets/169218266/5283874f-a125-4e64-8552-5ab072e2d965)

**Reply by blaisb** [reply]
Dear Majid,
Lethe supports the use of solid objects, which are rigid bodies with 6DOF motion that can interact with particles. If you are interested of modeling the deformation of the solid object, then this is not supported at the present time, you would need to add a linear elasticity (or another mechanics model) to the solid object walls

**Reply by Majid-MHZ** [reply]
Dear Bruno,

Thank you for your reply. Based on my understanding, the current condition of this software does not allow us to do this project and needs some modifications. Am I right?

**Reply by blaisb** [reply]
If you want either the particles to deform or the solid to deform, then yes modifications (some very important) would be required :)
